library/src/main/java/com/google/android/exoplayer/extractor/mp4/AtomParsers.java

```diff
@@ -114,10 +113,10 @@ public static TrackSampleTable parseStbl(Track track, Atom.ContainerAtom stblAto
     Atom.LeafAtom cttsAtom = stblAtom.getLeafAtomOfType(Atom.TYPE_ctts);
     ParsableByteArray ctts = cttsAtom != null ? cttsAtom.data : null;
 
+    int fixedSampleSize = sampleSizeBox.getFixedSampleSize();
```

Comment:
It looks like fixedSampleSize could be easily pushed into StszSampleSizeBox. There's no need for this method to know about it directly.